mrp_production_project_estimated_cost/wizard/wiz_create_fictitious_of.py

```diff
@@ -34,15 +35,16 @@ def do_create_fictitious_of(self):
                     'location_src_id': production_obj._src_id_default(),
                     'location_dest_id': production_obj._dest_id_default(),
                     'active': False,
-                    'product_uom': product.uom_id.id
+                    'product_uom': product.uom_id.id,
+                    'project_id': self.project_id.id
                     }
             new_production = production_obj.create(vals)
+            new_production.action_compute()
             production_list.append(new_production.id)
         if self.load_on_product:
-            for production_id in production_list:
+            for production in production_list:
                 try:
                     production = production_obj.browse(production_id)
-                    production.action_compute()
```

Comment:
Pero no es posible eso, al final estoy creando yo la OF. No hay posibilidad de que se ejecute esa función, desde la creación a cuando la ejecuto.